posthog/cdp/templates/rudderstack/test_template_rudderstack.py

```diff
@@ -0,0 +1,107 @@
+from inline_snapshot import snapshot
+from posthog.cdp.templates.helpers import BaseHogFunctionTemplateTest
+from posthog.cdp.templates.rudderstack.template_rudderstack import template as template_rudderstack
+
+
+class TestTemplateRudderstack(BaseHogFunctionTemplateTest):
+    template = template_rudderstack
+
+    def _inputs(self, **kwargs):
+        inputs = {
+            "host": "https://hosted.rudderlabs.com",
+            "token": "asjdkfasdkjfaskfkjfhdsf",
+            "identifier": "a08ff8e1-a5ee-49cc-99e9-564e455c33f0",
```

Comment:
@benjackwhite I can't seem to use `{person.uuid}` here. I just hardcoded the same person UUID instead.

Reply:
Yeah the templating is done in plugin-server so for these tests you are just doing the expected outcome of the template